scripts/arangodb-bash-setup/arangodb-bash-setup.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+set -e
+
+# IMPORTANT: Consult sample.env in apps/api/src/app/config for 
+# environment configuration prior to running this script
+
+# Check if this bash setup script is being run within it's own directory
+VALID_EXECUTE_DIRECTORY="coscrad/scripts/arangodb-bash-setup"
+CURRENT_PATH="$PWD";
+echo $'\nCURRENT_PATH: '$CURRENT_PATH;
+
+
+# Split path on /
+OLDIFS="$IFS";
+IFS='/';
+read -a strarr <<< $CURRENT_PATH;
+IFS="$OLDIFS";
+length=${#strarr[@]};
+p1=${strarr[$(($length-3))]};
+p2=${strarr[$(($length-2))]};
+p3=${strarr[$(($length-1))]};
+current_directory="${p1}/${p2}/${p3}";
+echo $'\nCurrent Directory: '$current_directory;
+
+if  [ $current_directory != $VALID_EXECUTE_DIRECTORY ];
+then
+  echo $'\n >> Please execute this script within its enclosing directory';
```

Comment:
I missed why you are required to run it in a certain directory. Could you explain?